### ISSUE TITLE
inference comeout some error, in DeepDPM_load_from_checkpoint.py 

### ISSUE THREAD

**Issue by huitang96** [BGU-CS-VIL/DeepDPM#42]
![2023-02-14 15-30-39 的屏幕截图](https://user-images.githubusercontent.com/66056602/218669022-85f2fbcd-a5a9-4910-b5fa-b7cc55ec6f6e.png)

Thank you for your great work, I am very interested in your work,  I followed the hints for training and generated an .pth.tar file. and have a little problem in DeepDPM_load_from_checkpoint.py  as shown in the picture, I would appreciate it if you could take the time to answer it.

**Comment by huitang96** [comment]
@meitarronen Would it be possible for you to kindly assist me in clarifying a question that I have? I would greatly appreciate any insights you may be able to provide.

**Comment by meitarronen** [comment]
Hey @huitang96! thank you for your interest!
This function manually sets all the argument that you supply in the saved model state.
It seems from the error code that the arguments you are using to load a model from checkpoint are not the same as the arguments you used to train the original model (which was saved).
You either have trained the model (of which the checkpoint was saved) using a seed, and in your arguments you didn't add the seed, or vice versa (you now added a seed to your arguments and you didn't train with it before).
If it was your intent, you can simply add a condition before the "setattr" and skip this field if the key equals to it.
Let me know if this helped!

**Comment by huitang96** [comment]
@meitarronen I just wanted to take a moment to thank you for all the help and guidance you have provided me with. With your code and direction, I was able to inherit and rewrite function functionalities, leading to successful training and prediction of my model.

However, I have encountered some difficulties with training my own dataset, which consists of 16 categories with approximately 160 images per category. The loss has been quite high（>2000）, with the use of RTX2060. I was wondering if you could offer any advice on which hyperparameters to adjust or other strategies that may be effective in improving the training efficiency for my dataset.

Thank you again for your invaluable assistance.
